languages/de.json

```diff
@@ -50,8 +50,8 @@
   "Text Replacement" : "Text ersetzen",
   "Transformations" : "Transformationen",
   "Undo" : "Widerrufen",
-  "Use Selection for Find" : "Auswahl suchen",
-  "Use Selection for Replace" : "Auswahl für Ersetzen übernehmen",
+  "Use Selection for Find" : "In Auswahl suchen",
+  "Use Selection for Replace" : "In Auswahl ersetzen",
```

Comment:
Again, according to Apple's Pages ``"Auswahl für Ersetzen übernehmen"`` should be right.